e2e/suite_test.go

```diff
@@ -51,12 +51,27 @@ func Test_All(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	if err := createClusterParamsDir(); err != nil {
+		t.Fatal(err)
+	}
+
+	t.Logf("dumping cluster parameters to local directory: %s", clusterParamsDir)
+	if err := dumpFileMapToDir(clusterParamsDir, clusterParams); err != nil {
+		t.Log("error dumping cluster parameters")
+		t.Error(err)
+	}
+
 	baseConfig, err := getBaseBootstrappingConfig(ctx, t, cloud, suiteConfig, clusterParams)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	for name, tc := range cases {
+		caseLogsDir, err := createVMLogsDir(name)
```

Comment:
this is outside t.Run() so it fails the whole suite, not a single case, probably should be per case

Reply:
derp